src/api.ts

```diff
@@ -0,0 +1,185 @@
+const base = 'http://localhost:3000';
+
+const garage = `${base}/garage`;
+const engine = `${base}/engine`;
+const winners = `${base}/winners`;
+
+export const getCars = async (
+    page: number,
+    limit = 7
+): Promise<{
+    items: { id: number; color: string; name: string }[];
+    count: string;
```

Comment:
It's recommended to use number type for fields like count, carsCount, and winnersCount